docs/tutorials/using_apis/connectivity/tls_tutorial.md

```diff
@@ -0,0 +1,173 @@
+<h1 id="tls-tutorial">TLS Tutorial</h1>
+
+Since Mbed OS 5.11, the [IP networking interface](../reference/ip-networking.html) has been extended to include [TLSSockets](../apis/tlssocket.html), which behave similarly to normal TCP sockets but automatically use Mbed TLS to set up a TLS connection to the server. This tutorial, based on [our blog entry](https://os.mbed.com/blog/entry/Adding-TLS-Sockets-to-Mbed-OS/), will help you understand and use TLS encryption in Mbed OS.
```

Comment:
Can you expand on this a little? When would you use this tutorial? When you want to set up a TLS connection to the server? When is that? Could you please provide a little more context?

Reply:
I added a sentence with a short explanation of the purpose of using TLS and gave an example (connecting to cloud).